### ISSUE TITLE
Python 3.9: AttributeError in core.py

### ISSUE THREAD

**Issue by grddavies** [sybrenstuvel/flickrapi#129]
Python version 3.9

Traceback:
```
File "... /lib/python3.9/site-packages/flickrapi/core.py", line 690, in data_walker
    photoset = rsp.getchildren()[0]
AttributeError: 'xml.etree.ElementTree.Element' object has no attribute 'getchildren'
````

As of python 2.7 and 3.9 `xml.etree.ElementTree.Element.getchildren()` has been removed, causing an error in line 690 of `core.py`

`photoset = rsp.getchildren()[0]`

https://docs.python.org/2/library/xml.etree.elementtree.html#xml.etree.ElementTree.Element.getchildren

**Comment by tedd4u** [comment]
I sent a PR to fix this … #130

**Comment by randomcascade** [comment]
Resolved as of #130.
